src/config.c

```diff
@@ -3116,6 +3116,7 @@ standardConfig static_configs[] = {
     /* Long Long configs */
     createLongLongConfig("busy-reply-threshold", "lua-time-limit", MODIFIABLE_CONFIG, 0, LONG_MAX, server.busy_reply_threshold, 5000, INTEGER_CONFIG, NULL, NULL),/* milliseconds */
     createLongLongConfig("cluster-node-timeout", NULL, MODIFIABLE_CONFIG, 0, LLONG_MAX, server.cluster_node_timeout, 15000, INTEGER_CONFIG, NULL, NULL),
+    createLongLongConfig("cluster-ping-interval", NULL, MODIFIABLE_CONFIG | DEBUG_CONFIG, 0, LLONG_MAX, server.cluster_ping_interval, 0, INTEGER_CONFIG, NULL, NULL),
```

Comment:
i suppose this makes it a major decision?

Reply:
I did mean hidden config! I got them confused

Reply:
Did we have hidden configs count as major features in the past? I would almost be inclined to set it up so we require a special build flag to build hidden configs so that people can't take dependencies on them unless they go out of there way.

Reply:
i rather avoid testing a special build of redis (and not testing the production one).
maybe we can rely on `enable-debug-command` to expose these configs, or better yet, just make this a debug command like `set-active-expire`.

if not, then it's simpler to just get quorum on adding this config.

Reply:
> i rather avoid testing a special build of redis (and not testing the production one).
maybe we can rely on enable-debug-command to expose these configs, or better yet, just make this a debug command like set-active-expire.

Debug options are not really better, since they can't be set as part of startup easily.

> if not, then it's simpler to just get quorum on adding this config.

Ok, added major tag.